frontend/README.md

```diff
@@ -18,15 +18,16 @@ Write a frontend application in TypeScript that Fetches and displays
    2. Can sort asc/desc by `Investor Annual Interest Due`
```

Comment:
Should this be a, b, c?

Reply:
Can't do a,b,c but can include un-ordered list in an ordered list. Looks a bit better 👍